### PR TITLE
Feature/storage/dm parallel transfer

### PR DESCRIPTION
**Pull request by amnguye**
Creating a PR to look over the API of DMLib

This excludes a lot of tests for now.

### CODE REVIEW

**Comment by amnguye** [comment]
Give me a second I have a lot of conflicts to resolve..

File: sdk/eventhub/Azure.Messaging.EventHubs.Experimental/api/Azure.Messaging.EventHubs.Experimental.netstandard2.0.cs [hunk 1]
```diff
@@ -0,0 +1,58 @@
+namespace Azure.Messaging.EventHubs.Producer
```
Comment (jsquire) [inline sdk/eventhub/Azure.Messaging.EventHubs.Experimental/api/Azure.Messaging.EventHubs.Experimental.netstandard2.0.cs:1]: Please revert the Event Hubs content.

File: sdk/storage/.editorconfig [hunk 2]
```diff
@@ -0,0 +1,136 @@
+# Rules in this file were initially inferred by Visual Studio IntelliCode from the C:\azure-sdk-for-net\sdk\storage codebase based on best match to current usage at 12/10/2021
```
Comment (amnguye) [inline sdk/storage/.editorconfig:1]: I dont even know how I added this. sigh let me fix this.

Comment (jsquire) [reply sdk/storage/.editorconfig:1]: much appreciated.

File: sdk/storage/Azure.Storage.Blobs/tests/ServiceClientTests.cs [hunk 3]
```diff
@@ -60,7 +60,7 @@ public void Ctor_Uri()
         {
             var accountName = "accountName";
             var accountKey = Convert.ToBase64String(new byte[] { 0, 1, 2, 3, 4, 5 });
-            var blobEndpoint = new Uri("http://127.0.0.1/" + accountName);
+            var blobEndpoint = new Uri("http://account.dfs.core.gov.net" + accountName);//new Uri("http://127.0.0.1/" + accountName);
```
Comment (amnguye) [inline sdk/storage/Azure.Storage.Blobs/tests/ServiceClientTests.cs:63]: TODO: undo change

**Pull request closed without merging**